src/lib.rs

```diff
@@ -221,3 +221,10 @@ impl IntoRawFd for Iface {
         self.fd.into_raw_fd()
     }
 }
+
+impl Drop for Iface {
```

Comment:
@vorner I tried to fixed it. But the compiler says:

```
error[E0509]: cannot move out of type `Iface`, which implements the `Drop` trait
   --> src/lib.rs:221:9
    |
221 |         self.fd.into_raw_fd()
    |         ^^^^^^^
    |         |
    |         cannot move out of here
    |         move occurs because `self.fd` has type `std::fs::File`, which does not implement the `Copy` trait
```